README.md

```diff
@@ -1,3 +1,6 @@
-# Samba-GlusterFS integration CI tests
+# Samba-Gluster integration test setup for centos-ci
 
-Continuous Integration helpers for Samba/CTDB + GlusterFS.
+This project contains the code to trigger the the samba-integration tests from
+the centos-ci. These are scripts to prepare a base-installed centos machine by
+installing the the required rpms, and then checking out the actual
+samba-integration repository an triggering the tests.
```

Comment:
s/an/and ?

Reply:
Thanks for catching. Updating... BTW, you can use "request changes" in the review dialog.